lib/service/Service.ts

```diff
@@ -0,0 +1,86 @@
+import assert from 'assert';
+import { GrpcComponents } from '../grpc/GrpcServer';
+import Logger from '../Logger';
+import Pool from '../p2p/Pool';
+import OrderBook from '../orderbook/OrderBook';
+import { orders, matchingEngine } from '../types';
+import LndClient from '../lndclient/LndClient';
+import RaidenClient, { TokenSwapPayload } from '../raidenclient/RaidenClient';
+import grpc from 'grpc';
+import { PairInstance } from '../types/db';
+import { GetInfoResponse } from '../proto/lndrpc_pb';
+import { Orders } from '../orderbook/OrderBookRepository';
+import { OwnOrder } from '../types/orders';
```

Comment:
few unused imports over here

Reply:
Thanks. I'm trying to get `no-unused-variable` into our base style guide to catch things like this.